setup.py

```diff
@@ -38,7 +38,7 @@ def get_description():
     return data
 
 
-REQUIREMENTS = ['swmm-toolkit==0.8.2', 'julian==0.14']
+REQUIREMENTS = ['swmm-toolkit==0.9.0', 'julian==0.14', 'aenum==3.1.11']
```

Comment:
@abhiramm7, it's no big deal now.  In the past we used parse the `requirements.txt` file into a python list here.

Reply:
oh for sure, I can update it :)

Reply:
Hey @bemcdonnell,

 I was about to update the setup.py to read from requirements.txt, but StackOverflow told me we should keep them separate. Apparently, there is a reason why there are two things. I will update setup.py and requirements.txt to reflect how the gods of python intended them. 
 
> So the takeaway:
> 
> `setup.py` should declare the loosest possible dependency versions that are still workable. Its job is to say what a particular package can work with.
> `requirements.txt` is a deployment manifest that defines an entire installation job, and shouldn't be thought of as tied to any one package. Its job is to declare an exhaustive list of all the necessary packages to make a deployment work.
> Because these two things have such different content and reasons for existing, it's not feasible to simply copy one into the other.
>  

 Ref: https://stackoverflow.com/questions/14399534/reference-requirements-txt-for-the-install-requires-kwarg-in-setuptools-setup-py and https://packaging.python.org/en/latest/discussions/install-requires-vs-requirements/